PopupDialog/Classes/TransitionAnimator.swift

```diff
@@ -52,7 +52,7 @@ internal class TransitionAnimator: NSObject, UIViewControllerAnimatedTransitioni
             to = transitionContext.viewControllerForKey(UITransitionContextToViewControllerKey)!
             from = transitionContext.viewControllerForKey(UITransitionContextFromViewControllerKey)!
             let container = transitionContext.containerView()
-            container.addSubview(to.view)
+            container!.addSubview(to.view)
```

Comment:
Unit tests fail as of force unwrapping container.

> cannot force unwrap value of non-optional type 'UIView'

Any particular reason for force unwrapping here?